docs/administration/configuration/configuring-workspace-retention-rules/index.md

```diff
@@ -0,0 +1,34 @@
+# Configuring workspace retention rules
+
+REANA allows users to keep files in their workflows for a limited time using **retention rules**.
```

Comment:
As of version 0.9, REANA administrators can configure a global workflow file retention period after which all the files produced by user workflows that are not declared as "inputs" or "outputs" will be automatically deleted after a certain number of days. (By default, every file is kept indefinitely.)

If you would like to limit the workspace file retention period globally, you can configure workspaces.retention_rules.maximum_period ` Helm value to the desired amount of days, such as 3650. This will make sure that all files produced by users will be kept at most for ten years. Note that users will be able to define their individually desired retention rules (link) to a period not exceeding this global maximum.

We recommend to keep the `workspaces.retention_rules.maximum_period` value to its default value `forever` and  rather use the user disk storage resource quotas (link) to limit the amount of storage that each user can consume. (This is the default REANA behaviour.) 

Note that if you redeploy REANA and modify its `workspaces.retention_rules.maximum_period` settings between two rolling deployments, e.g. from "forever" to "3650 days", the new workflows will use the updated global maximum retention period (ten years) while the old workflows will keep using older default (forever).

Reply:
I have removed `after a certain number of days` to avoid the repetition with `... after which ...`.
I have also removed `(This is the default REANA behaviour.)` as we were already saying `... its default value ...`.